robottelo/ui/computeprofile.py

```diff
@@ -0,0 +1,28 @@
+# -*- encoding: utf-8 -*-
+from robottelo.ui.base import Base, UINoSuchElementError
+from robottelo.ui.locators import locators
+from robottelo.ui.navigator import Navigator
+
+
+class ComputeProfile(Base):
+    """Provides the CRUD functionality for Compute Profiles."""
+
+    def search(self, name):
+        """Search for existing compute profile from UI."""
+        Navigator(self.browser).go_to_compute_profiles()
+        self.wait_for_ajax()
```

Comment:
This wait_for_ajax is something that we can think as a future improvement, the navigator should take care of this.